rust/rpc/src/lib.rs

```diff
@@ -173,7 +174,7 @@ impl<W:Write> RpcPeer<W> {
             .insert("method", method)
             .insert("params", params)
             .unwrap()) {
-            print_err!("send error on send_rpc_async method {}: {}", method, e);
+            print_err!("send error on send_rpc_sync method {}: {}", method, e);
```

Comment:
Btw, to reduce the commit to the actual change, I am fine with updating the PR to not contain this change and also not the removed trailing spaces in 112